sqlstore/src/main/java/uk/ac/bbsrc/tgac/miso/db/migration/V0251__StatusXmlToRunTable.java

```diff
@@ -43,7 +45,8 @@ public void migrate(Connection connection) throws Exception {
 
   private Map<String, String> getStatusXml(Connection connection) throws SQLException {
     Map<String, String> result = new HashMap<>();
-    PreparedStatement statement = connection.prepareStatement("SELECT s.runName, s.xml FROM Status AS s WHERE s.xml IS NOT NULL");
+    PreparedStatement statement =
```

Comment:
Is it ok to modify an existing migration? Looks like it's just formatting so maybe not needed?

Reply:
It is just formatting, which doesn't matter since Java migrations are not validated by checksum